Include/internal/pycore_frame.h

```diff
@@ -52,13 +47,16 @@ typedef struct _PyInterpreterFrame {
     PyCodeObject *f_code; /* Strong reference */
     PyFrameObject *frame_obj; /* Strong reference, may be NULL */
     struct _PyInterpreterFrame *previous;
-    int f_lasti;       /* Last instruction if called */
+    _Py_CODEUNIT *next_instr;
     int stacktop;     /* Offset of TOS from localsplus  */
     bool is_entry;  // Whether this is the "root" frame for the current _PyCFrame.
     char owner;
     PyObject *localsplus[1];
 } _PyInterpreterFrame;
 
+#define _PyInterpreterFrame_LASTI(IF) \
+    ((int)((IF)->next_instr - _PyCode_CODE((IF)->f_code) - 1))
+
```

Comment:
The i in lasti referred to 'instruction', but now it's the previous word which could be in the cache of that instruction, so should this macro be named something else?

Reply:
I mainly named it `LASTI` to make its meaning clear for anyone familiar with the prior code. Also, the `f_lasti` member of Python-layer frame objects has the same value (and we're not renaming that), so to me it makes sense to keep the names in sync.

I'll add a clarifying comment, though.